UKHomeOffice/sgmr-service
# SPCR-491 - Make only spaces invalid

### AC / Description of changes
When completing a voyage plan, if User selects 'Yes' to the questions 'Are there any prohibited or restricted goods on board?'/ 'Does the vessel have any duty-free stores?'

They are presented with a text area that must be filled to continue, if the user enters a 'space' the validation of the area accept this can can continue.

Check all our text inputs and make only spaces a falsey value

### To Test
- Go to any form in the application with a text input e.g. (Customs page w/ text boxes, skipper page, PC form etc.)
- Enter a space or string of spaces
- > The input should error unless there are other characters inputted

### Notes

---

**Remember to merge to feature branches not `master`.**
